egs2/dns_ins20/enh1/local/dns_create_mixture.sh

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
+
+configure=   # Path to the configure file
+
+. utils/parse_options.sh
+. path.sh
+. cmd.sh
+
+if [ $# -ne 2 ]; then
+  echo "Usage: $0 --configure <configure> <dns> <dns_wav> "
+  echo " where <dns> is dns directory,"
+  echo " <dns_wav> is wav generation space."
+  exit 1;
+fi
+
+dns=$1
+dns_wav=$2
+
+rm -r data/ 2>/dev/null || true
+mkdir -p data/
+
+mod_conf=true
+if [ -z "$configure" ]; then
```

Comment:
I guess there should be a `else` branch to handle the case where `configure` is passed as an argument.

Reply:
Thanks for your suggestions. This part modifies the default configuration file provided by DNS. When the user specifies the configure, the script keeps it the same. I will add this information into the explanation of the optional argument.

Reply:
Okay, but some variables are only defined in the `if` branch, like `train_cfg ` in https://github.com/espnet/espnet/blob/be73d0de071e9a7fcaf98ad2e5c94dad9ca73cda/egs2/dns_ins20/enh1/local/dns_create_mixture.sh#L65

Reply:
Thanks, I fixed it!